packages/core/src/amazonq/webview/ui/storages/tabsStorage.ts

```diff
@@ -4,7 +4,7 @@
  */
 
 export type TabStatus = 'free' | 'busy' | 'dead'
-export type TabType = 'cwc' | 'featuredev' | 'unknown'
+export type TabType = 'cwc' | 'featuredev' | 'gumby' | 'unknown'
```

Comment:
Note: Here `TabType` is a string literal. Ignore previous comments on string literal.